content/en/docs/contribute/style/style-guide.md

```diff
@@ -146,6 +146,33 @@ Set the value of the `replicas` field in the configuration file. | Set the value
 The value of the `exec` field is an ExecAction object. | The value of the "exec" field is an ExecAction object.
 {{< /table >}}
 
+### Use code style for Kubernetes command tool and component names
+
+{{< table caption = "Do and Don't - Use code style for Kubernetes command tool and component names" >}}
+Do | Don't
+:--| :-----
+The `kubelet` preserves node stability. | The kubelet preserves node stability.
+The `kube-proxy` maintains network rules on nodes. | The kube-proxy maintains network rules on nodes.
+Kubectl handles locating and authenticating to the API server. | `kubectl` handles locating and authenticating to the apiserver.
+{{< /table >}}
+
+### Starting a sentence with a component tool or component name
+
+{{< table caption = "Do and Don't - Starting a sentence with a component tool or component name" >}}
+Do | Don't
+:--| :-----
+The `kube-apiserver` is designed to scale horizontally. | `kube-apiserver` is designed to scale horizontally.
+Kubectl handles locating and authenticating to the API server. | The Kubectl handles locating and authenticating to the apiserver.
```

Comment:
This one fails the guide before this one ... :)

Reply:
and the columns should be swapped?

Reply:
OK. So 1) Do not start a sentence with a tool/component name (see my comment above)
2) Line 165 -- Swap the column entries. Is it acceptable to use `apiserver` vs `kube-apiserver`?

Reply:
The component is called `kube-apiserver`. I really dislike the `apiserver` form because it comes from nowhere.